Jenkinsfile

```diff
@@ -1,178 +1,111 @@
-@Library('xmos_jenkins_shared_library@v0.18.0') _
+// This file relates to internal XMOS infrastructure and should be ignored by external users
+
+@Library('xmos_jenkins_shared_library@v0.39.0') _
 
 getApproval()
 
 pipeline {
-  agent none
-
-  environment {
-    REPO = 'lib_spi'
-    VIEW = getViewName(REPO)
+  agent {
+    label 'x86_64 && linux'
   }
-
   options {
+    buildDiscarder(xmosDiscardBuildSettings())
     skipDefaultCheckout()
+    timestamps()
+  }
+  parameters {
+    choice(name: 'TEST_LEVEL', choices: ['smoke', 'default', 'extended'],
+            description: 'The level of test coverage to run'
+    )
+    string(
+      name: 'TOOLS_VERSION',
+      defaultValue: '-j -b markp_xsim_expose_signals_from_usb_shim latest',
```

Comment:
I think this is a copy paste issue from lib_xud that currently needs a dev build of tools.

Reply:
oops